components/wysiwyg/widget-block-edition/index.js

```diff
@@ -0,0 +1,87 @@
+import React, { useReducer, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import debounce from 'lodash/debounce';
+
+// services
+import { fetchWidgets } from 'services/widget';
+
+// store
+import { connect } from 'react-redux';
```

Comment:
No need of a special `store` section, it's just another dependency, move it above. Actually, it's not the `store` itself, so it can even be misleading in some cases.